app/Http/Controllers/SystemController.php

```diff
@@ -62,6 +64,39 @@ public function facilityList(): JsonResource
         return FacilityResource::collection(Facility::query()->get());
     }
 
+    #[OA\Get(
```

Comment:
jak coś, żeby działąło upranienie, to nie wystarczy openapi, tylko jeszcze trzeba override'ować
```
protected function initPermissions(): void
```

i to nie w tym kontrolerze, ten ma tylko publiczne endpointy z ogólną konfiguracją systemu

może Controllers/Facility/MeetingController

Reply:
Zrobione.